tips/TIP-0020/tip-0020.md

```diff
@@ -0,0 +1,489 @@
+    * The referenced _Unlock_ unlocks the alias defined by the unlocking address of the UTXO.
+  * If it is an _NFT Unlock_:
+    * The address unlocking the UTXO must be a _NFT Address_.
+    * The referenced _Unlock_ unlocks the NFT defined by the unlocking address of the UTXO.
+
+If a _Transaction Payload_ passes the semantic validation, its referenced UTXOs must be marked as spent and its new outputs must be created/booked in the ledger. The _Block ID_ of the block encapsulating the processed payload then also becomes part of the input for the White-Flag Merkle tree hash of the confirming milestone ([TIP-4](../TIP-0004/tip-0004.md)).
+
+Transactions that do not pass semantic validation are ignored. Their UTXOs are not marked as spent and their outputs are not booked in the ledger.
+
+## Miscellaneous
+
+### Transaction timestamps
+
+Since transaction timestamps – whether they are signed or not – do not provide any guarantee of correctness, they have been left out of the _Transaction Payload_. Instead, the global timestamp of the confirming milestone ([TIP-8](../TIP-0008/tip-0008.md)) is used.
+
+### Address reuse
+
+While, in contrast to Winternitz one-time signatures (W-OTS), producing multiple Ed25519 signatures for the same private key and address does not decrease its security, it still drastically reduces the privacy of users. It is thus considered best practice that applications and services create a new address per deposit to circumvent these privacy issues.
+
+In essence, Ed25519 support allows for smaller transaction sizes and to safely spend funds which were sent to an already used deposit address. Ed25519 addresses are not meant to be used like email addresses. See this [Bitcoin wiki article](https://en.bitcoin.it/wiki/Address_reuse) for further information.
+
+# Drawbacks
+
+* The new transaction format is the core data type within the IOTA ecosystem. Changing it means that all projects need to accommodate it, including wallets, web services, client libraries and applications using IOTA in general. It is not possible to keep these changes backwards compatible, meaning that all nodes must upgrade to further participate in the network.
+* It is not possible to produce a valid transaction without having access to the content of the consumed outputs.
+
+# Rationale and alternatives
+
+* _Inputs Commitment_ and _Network ID_ are both explicit fields of the transaction, while they could be made configuration
+  parameters for the signature generating process. In this scenario the signature would be invalid if the parameters on client
+  and network side mismatch. While this would reduce the size of a transaction, it would make it impossible to debug the
+  reason for having an invalid signature and transaction. With the current solution we intend to optimize for ease of development.
+* Uniqueness of all inputs is kept as it prevents introducing double spends in the same transaction.
```

Comment:
```suggestion
```
I think that sentence should just be removed, right?

Reply:
In syntactic validation we have this rule:
> Each pair of Transaction ID and Transaction Output Index must be unique in the list of inputs.

Therefore, this sentence is still valid, no?

Reply:
OK, fair enough